portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACPagePath.macro

```diff
@@ -0,0 +1,54 @@
+definition {
+
+	@summary = "Clears segment search filter"
+	macro clearSegmentSearchFilter() {
+		Type.sendKeys(
+			locator1 = "ACSettings#SEARCH_BAR",
+			value1 = "keys=CONTROL,a");
+
+		KeyPress(
+			locator1 = "ACSettings#SEARCH_BAR",
+			value1 = "\BACK_SPACE");
+	}
+
+	@summary = "Selects the segment in page filter"
+	macro clickFilterSegment(segmentName = null) {
```

Comment:
Hey @joshitagaki, I think the `segmentName` variable is always necessary to use this new macro, right? So I think we can set it without `null` here so it can be a required variable

```suggestion
	macro clickFilterSegment(segmentName) {
```